catalog/policies/git_push.proposed-run_test.rego

```diff
@@ -0,0 +1,134 @@
+package spacelift
+
+test_propose_project_root_affected {
+  propose with input as {
+    "stack": {
+      "project_root": "components/terraform/ecr",
+      "labels": [],
+    },
+    "pull_request": {
+      "action": "opened",
+      "draft": false,
+      "labels": [],
+      "diff": ["components/terraform/ecr/main.tf"],
+    },
+  }
+}
+
+test_propose_stack_config_affected_imports_label {
+  propose with input as {
+    "stack": {
+      "project_root": "components/terraform/ecr",
+      "labels": ["import:stacks/catalog/ecr.yaml"],
```

Comment:
we use just the `deps:` labels in Spacelift stacks, so this is nice to have but will never be triggered

Reply:
what are we testing by providing `stack.labels`?

Reply:
https://github.com/cloudposse/terraform-spacelift-cloud-infrastructure-automation/blob/ea74791a744069566b5fd41456caf71be2ec05ba/catalog/policies/git_push.proposed-run.rego#L73-L77

Reply:
If this code is unused and doesnt need to be tested, should we remove the import label check from the policy itself?

Reply:
yes, let's remove it, it's not in use

Reply:
on second thought, could we do this in a follow up PR ?

At the moment this PR is only adding a complete test so it can be a `no-release`. A change to the logic of the original policy would mean that we could be removing cruft but also breaking something.